### DISCUSSION TITLE
Errors

### DISCUSSION THREAD

**Discussion by nobaraaintdead** [Aelto/W3_RandomEncounters_Tweaks#156] [General]
Error [mod0000_mergedfiles]game\gameplay\damage\damagemanagerprocessor.ws(1977): syntax error, unexpected TOKEN_IF, expecting TOKEN_FUNCTION, near 'if'
Error [mod0000_mergedfiles]game\npc\npc.ws(1854): syntax error, unexpected TOKEN_IDENT, expecting TOKEN_FUNCTION, near 'Scaling'
Error [mod0000_mergedfiles]game\player\r4player.ws(16102): Unexpected end of file found after '{' at line 7

I got these errors after following the in-depth guide. I don't know what I did wrong.

**Comment by Aelto** [comment]
Hello,

All the merging instructions can be found [here](https://aelto.github.io/tw3-random-encounters-reworked/indepth-guide/#part-5), everything important is written here. The rest are basic merging errors i cannot solve (or else i wouldn't have free time anymore). 

A general advice when merging sharedutils code: On each conflict picking sharedutils first then the other mod will often solve the conflict. You should always pick the sharedutils code and the other mod's code, what matters is the order

**Reply by nobaraaintdead** [reply]
![THE WITCHER 3](https://user-images.githubusercontent.com/118868409/207537535-09bfb69d-efa2-43af-9b81-0c106be908db.jpg)
Any suggestions?
